src/Components/Notifications/NotificationsPills.tsx

```diff
@@ -1,17 +1,30 @@
 import { ActionType } from "@artsy/cohesion"
-import { Flex, Pill } from "@artsy/palette"
+import { Flex, Pill, Skeleton, SkeletonBox } from "@artsy/palette"
 import { NotificationType } from "Components/Notifications/types"
 import { useNotificationsContext } from "Components/Notifications/useNotificationsContext"
+import { Suspense } from "react"
+import { graphql, useLazyLoadQuery } from "react-relay"
 import { useTracking } from "react-tracking"
+import { NotificationsPillsQuery } from "__generated__/NotificationsPillsQuery.graphql"
+import { times } from "lodash"
 
 export const NOTIFICATIONS_PILLS = [
   { value: "All", name: "all" },
+  { value: "Offers", name: "offers" },
   { value: "Alerts", name: "alerts" },
   { value: "Following", name: "following" },
 ]
+
 export const NotificationsPills: React.FC = () => {
   const { trackEvent } = useTracking()
   const { setCurrentNotificationFilterType, state } = useNotificationsContext()
+  const pills = useLazyLoadQuery<NotificationsPillsQuery>(
```

Comment:
We should avoid `useLazyLoadQuery` in favor of `useClientQuery` (from utils/hooks). That comes with its own loading state, and we get to skip a lot of future complexity around SSR and this relay hook. cc @anandaroop